beacon-chain/rpc/eth/beacon/handlers_pool.go

```diff
@@ -27,6 +28,8 @@ import (
 	"go.opencensus.io/trace"
 )
 
+const broadcastBLSChangesRateLimit = 128
```

Comment:
why is this not a constant in field params or somewhere like that?

Reply:
It's just a rate-limiting thing for one endpoint. I think it was added by Potuz to constrain the number of messages at Capella fork. No need to add it to field params as it's only used in one place.